cmd/influx/task.go

```diff
@@ -53,3 +55,312 @@ func newTaskCreateCmd() *cli.Command {
 		},
 	}
 }
+
+func newTaskFindCmd() *cli.Command {
+	var params task.FindParams
+	params.Limit = TaskMaxPageSize
```

Comment:
I think that instead of setting it here, you could set it as `Value` on the `cli.IntFlag` struct. That way it'll show up in the CLI help-text